scripts/extensions/ai-widget/src/translations/translations-body.tsx

```diff
@@ -0,0 +1,178 @@
+                    translatedFromLanguage: article.language,
+                });
+            });
+        }
+    }
+
+    render(): ReactNode {
+        const {error, loading, translation, article, generateTranslation} = this.props;
+        const {gettext} = superdesk.localization;
+
+        if (error) {
+            return (
+                <Spacer v alignItems="center" gap="8" justifyContent="center" noWrap>
+                    <Button
+                        style="hollow"
+                        onClick={generateTranslation}
+                        text={gettext('Regenerate')}
+                    />
+                    <Heading type="h6" align="center">
+                        {gettext('There was an error when trying to generate a translation.')}
+                    </Heading>
+                </Spacer>
+            );
+        }
+
+        if (loading) {
+            return <Loader overlay />;
+        }
+
+        if (translation == '') {
+            return (
+                <Spacer v alignItems="center" gap="8" justifyContent="center" noWrap>
+                    <Heading type="h3" align="center">
+                        {gettext('No Translations yet')}
+                    </Heading>
+                    <Heading type="h6" align="center">
+                        {gettext('Choose the language and click the translate button at the bottom')}
+                    </Heading>
+                </Spacer>
+            );
+        }
+
+        return (
+            <Container gap="small" direction="column" className="sd-overflow--y-auto">
+                <Spacer v gap="16" justifyContent="end" noGrow>
+                    <Spacer h gap="4" justifyContent="end" alignItems="center" noWrap>
+                        <Label
+                            text={this.props.mode === 'other'
+                                ? this.state.translatedFromLanguage
+                                : this.props.article.language
+                            }
+                            size="small"
+                        />
+                        <Icon
+                            name="arrow-right"
+                            size="small"
+                        />
+                        <Label
+                            text={this.props.activeLanguageId}
+                            size="small"
+                            type="primary"
+                        />
+                    </Spacer>
+                    <div dangerouslySetInnerHTML={{__html: translation}} />
+                </Spacer>
+                <ButtonGroup orientation="horizontal" align="center">
+                    <IconButton
+                        ariaValue={gettext('Copy')}
+                        icon="copy"
+                        onClick={() => {
+                            navigator.clipboard.writeText(translation);
+                        }}
+                    />
+                    {this.props.mode !== 'other' && (
```

Comment:
No, I'll fix it so it's 'current', it might've been because initial state is `null`. If that's the case I'll see if I can tweak the initial state so that it's 'current' by default, if not leave it open as is for anything but 'other'.